### ISSUE TITLE
In this sample project, the inspector doesn't show a controlled shadow

### ISSUE THREAD

**Issue by maltenuhn** [concrete-utopia/utopia#2033]
https://utopia.app/p/5b0c4ce1-tar-panda/

1. In view mode, click on one of the dots. It should be styled as "selected" (blue box shadow)
2. Select that dot in the navigator
3. look at the inspector

Expected: there's a controlled shadow section
Observed: there isn't

**Comment by seanparsons** [comment]
@maltenuhn If you edit the component, the div inside it has a controlled shadow, is that not correct?

**Comment by maltenuhn** [comment]
@seanparsons Yes, once you _edit_ the component, the inspector is correct. This ticket is about this divergence:

- if you edit/isolate the component and the select the outermost div it renders, it shows you the background set to '#FF00AA', the border set to `1px solid white`, _and the boxShadow as controlled_
- if you only _select_ the component (without editing/isolating it), you see the background correctly set to #ff00aa in the inspector, and the border correctly set to 1px solid white -  _but the shadow section is completely empty_

I believe that the shadow should also show up (as controlled) in the second case, hence this ticket.
